Commands/InteractionServiceHandler.cs

```diff
@@ -25,7 +26,7 @@ public static async Task InitAsync()
 		}
 		catch
```

Comment:
Provide additional information about the error
```suggestion
		catch (Exception e)
```